projects/storefrontlib/src/cms-components/organization/permissions/permission-form/permission-form.component.html

```diff
@@ -96,40 +133,10 @@
         </label>
       </div>
 
-      <div class="form-group">
-        <ng-container
-          *ngIf="businessUnits$ | async as businessUnits"
-          formGroupName="orgUnit"
-        >
-          <div *ngIf="businessUnits.length !== 0">
-            <label aria-required="true">
-              <span class="label-content required">{{
-                'permissionForm.businessUnits.label' | cxTranslate
-              }}</span>
-              <ng-select
-                formControlName="uid"
-                [searchable]="false"
-                [clearable]="false"
-                [items]="businessUnits"
-                bindLabel="name"
-                bindValue="id"
-                [class.invalid]="isNotValid('orgUnit')"
-              >
-                <!--                (change)="businessUnitSelected($event)"-->
-              </ng-select>
-              <ng-container
-                *ngIf="isNotValid('orgUnit')"
-                [ngTemplateOutlet]="validation"
-                [ngTemplateOutletContext]="{
-                  control: form.controls.orgUnit['controls']['uid']
-                }"
-              ></ng-container>
-            </label>
-          </div>
-        </ng-container>
-      </div>
-
-      <div class="form-group">
+      <div
+        class="form-group"
+        *ngIf="selectedPermissionType !== 'B2BBudgetExceededPermission'"
```

Comment:
I think, we have already info that fields are enabled. What do you think about usage our currencyControl.enabled flag?

The same for all of dynamic fields

Reply:
done